.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,41 @@
+name: Pull Request Checks
+
+on:
+  pull_request:
+    types: [opened]
+  pull_request_review:
+    types: [submitted]
+
+concurrency: pr-${{ github.ref }}
+
+jobs:
+  dependency-checks:
+    name: Dependency Checks
+
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Checkout Code
+        uses: actions/checkout@v2
+
+      - name: Set Branch Name
+        uses: nelonoel/branch-name@v1.0.1
```

Comment:
When you implement the Sonatype Dependncy check plugin, you will want to replace lines 21-22 with:

```
    - name: Set Branch Env Var
      run: | 
        echo "BRANCH_NAME=$(echo ${{github.event.pull_request.head.ref}} | tr / -)" >> $GITHUB_ENV
```

Reply:
Whoops, thank you! Will make that change.